### PR TITLE
fix docs for recreate strategy

### PR DESCRIPTION
**Pull request by dvagnero**
Fix information about recreate strategy and add a note about data loss

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]
LGTM. I'll wait for @Crumby's review before merging

**Review by Crumby — APPROVED** [review]

**Comment by ryanemerson** [comment]
Thanks @dvagnero

**Pull request merged**
